lib/node-utils.ts

```diff
@@ -17,6 +17,7 @@ export function isNewExpression(
   return node && node.type === 'NewExpression';
 }
 
+// TODO: remove this one and use ASTUtils one instead
```

Comment:
🧐

Reply:
oh man, I saw your `ASTUtils` now... ignore my previous comments. It's awesome they export that 🚀